python/cac_tripplanner/templates/base.html

```diff
@@ -3,12 +3,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>{% block title %}CAC Trip Planner{% endblock %}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0">
+    <title>{% block title %}Open Transit Planning for Bikes, Public Transit, and Walking | Clean Air Council{% endblock %}</title>
 
     {% block cssimports %}
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
```

Comment:
Either this or in a separate mixin file - I think the way you did it is best since every page we have is going to use it.